src/backends/custom/custom.h

```diff
@@ -249,6 +287,15 @@ TRTIS_CUSTOM_EXPORT int CustomExecute(
     void* custom_context, uint32_t payload_cnt, CustomPayload* payloads,
     CustomGetNextInputFn_t input_fn, CustomGetOutputFn_t output_fn);
 
+/// See CustomExecute.
```

Comment:
"Execute the custom model using the version 2 implementation of the execute interface. This function must be defined when the custom backend returns 2 from CustomVersion. See CustomExecute for description of the parameters."